tasks.py

```diff
@@ -26,8 +29,38 @@
 invoke_tasks.PROJECT_DIR = Path(__file__).resolve().parent
 
 
+@task
+def upgrade(ctx: Context):
```

Comment:
No need for this task anymore ... we use renovate now

Reply:
deleted task